support/azure/azure-kubernetes/availability-performance/node-not-ready-custom-script-extension-errors.md

```diff
@@ -49,9 +49,9 @@ Set up your custom Domain Name System (DNS) server so that it can do name resolu
 
 - Don't use the Azure DNS IP address with the IP addresses of your custom DNS server. Doing this isn't recommended.
 
-- Avoid using IP addresses instead of the DNS server in DNS settings. You can use Azure CLI commands to check for this situation on a virtual machine (VM) scale set or availability set.
+- Avoid using IP addresses instead of the DNS server in DNS settings. You can use Azure CLI commands to check for this situation on a Virtual Machine Scale Set (VMSS) or availability set.
```

Comment:
@jaewonparkalexander  VMSS is not an official abbreviation for Virtual Machine Scale Set. So I will removed it from the article.